src/resources/postcss/views/full/_single-event.pcss

```diff
@@ -231,11 +258,10 @@
 .tribe-events-meta-group {
 
 	.tribe-events-single-section-title {
-		@mixin heading;
-		@mixin heading-8;
-
-		color: var(--color-text-primary-light);
-		font-weight: var(--font-weight-regular);
+		color: var(--color-text-primary);
+		font-size: 15px;
```

Comment:
No judgement, but have to ask: Is there a good reason to vary this from existing sizes?
```
--font-size-2: 14px;
--font-size-3: 16px;
```

Reply:
asking Rachel, thanks

Reply:
we're going for 14px